components/Landing/QiskitCapabilitiesSection/LandingQiskitCapabilitiesSectionComponent.vue

```diff
@@ -92,64 +56,35 @@ const capabilities: QiskitCapability[] = [
     },
   },
   {
-    title: "Run on real hardware",
+    title: "Transpile to any hardware",
```

Comment:
```suggestion
    title: "Run on any hardware",
```
I think this would be better because we already use the word transpile for another heading and I think its a bit too repetitive to use it twice. Also I'm unsure that "transpile to" is good grammar. But maybe @1ucian0 disagrees?

Reply:
I agree, I'm gonna merge it and, if Luciano thinks it should be `Transpile`, we can always fix it in a new PR. I just don't want to block the merge with this